### PR TITLE
{data}[foss/2019b] bitarray v1.2.1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by stevecdavis**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.2
See https://gist.github.com/6409c3615b86b87102cfe6668b19e850 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/cb22c2c4c37fbba0b4e2e837470f75fd for a full test report.

File: easybuild/easyconfigs/b/bitarray/bitarray-1.2.1-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ilanschnell/bitarray'
+description = "bitarray provides an object type which efficiently represents an array of booleans"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['050cd30b810ddb3aa941e7ddfbe0d8065e793012d0a88cb5739ec23624b9895e']
```
Comment (verdurin) [inline easybuild/easyconfigs/b/bitarray/bitarray-1.2.1-foss-2019b-Python-3.7.4.eb:14]: @stevecdavis could you look again at the `checksum`, please?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/bitarray/bitarray-1.2.1-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (stevecdavis) [reply easybuild/easyconfigs/b/bitarray/bitarray-1.2.1-foss-2019b-Python-3.7.4.eb]: Sorry, not sure why the checksum was wrong.  I updated it and re-submitted it.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/c002dff4f766091e4dc77159722cfd7f for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.2
See https://gist.github.com/4805b0c2860bafcf509b5a3f14e9dbe9 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @stevecdavis!

**Pull request merged**